sbtc/src/rpc.rs

```diff
@@ -4,11 +4,14 @@ use std::num::NonZeroU8;
 
 use bitcoin::consensus;
 use bitcoin::consensus::Decodable as _;
+use bitcoin::Amount;
 use bitcoin::BlockHash;
+use bitcoin::Denomination;
 use bitcoin::Transaction;
 use bitcoin::Txid;
+use bitcoincore_rpc::json::EstimateMode;
 use bitcoincore_rpc::Auth;
-use bitcoincore_rpc::RpcApi;
+use bitcoincore_rpc::RpcApi as _;
```

Comment:
That's this for?

Reply:
This is to communicate that this imported thing is a trait that is necessary but isn't directly used. It's more of a stylistic change than anything.